src/components/canvas/Interaction/ContextMenu/useContextMenuItems.tsx

```diff
@@ -32,6 +32,29 @@ const zoomToFit = (boundsControls: BoundsApi) => {
   boundsControls?.refresh().reset().fit().clip()
 }
 
+const editAppearance = (drawingId: DrawingID, solidId: ObjectID) => {
```

Comment:
This is a duplicate of `react-cad`, right?

Reply:
Yes